bigtable-hbase-integration-tests/pom.xml

```diff
@@ -104,8 +104,8 @@ limitations under the License.
                 </dependency>
                 <dependency>
                     <groupId>io.netty</groupId>
-                    <artifactId>netty-tcnative</artifactId>
-                    <version>1.1.33.Fork9</version>
+                    <artifactId>netty-tcnative-boringssl-static</artifactId>
+                    <version>1.1.33.Fork13</version>
                     <classifier>${os.detected.classifier}</classifier>
```

Comment:
Is the classifier still needed?  I don't think it should be for what we were told.